packages/video_player/video_player/example/pubspec.yaml

```diff
@@ -37,3 +37,8 @@ flutter:
     - assets/bumble_bee_captions.srt
     - assets/bumble_bee_captions.vtt
     - assets/Audio.mp3
+
+# FOR TESTING ONLY. DO NOT MERGE.
```

Comment:
remove this

Reply:
Should I remove every `dependency_overrides`? Won't the tests fail?